applications/openshift/networking/routes_protected_by_tls/tests/all_route_with_one_null.pass.sh

```diff
@@ -0,0 +1,132 @@
+#!/bin/bash
+
+# remediation = none
+
+yum install -y jq
+
+kube_apipath="/kubernetes-api-resources"
+
+mkdir -p "$kube_apipath/apis/route.openshift.io/v1"
+
+routes_apipath="/apis/route.openshift.io/v1/routes"
+
+# This file assumes that we have two routes outside the openshift/kube namespaces, and 
```

Comment:
nit: this comment is unrelated to the tests in this patch, right?

Reply:
that's right, I am going to remove this one